src/server/app.js

```diff
@@ -196,10 +197,19 @@ io.sockets.on('connection', function (socket) {
           socket.emit('update-feedback-interval', [resultNeg, resultPos])
         })
       })
-      // Get last 10 messages
-      messagesController.getLastTen(lecture).then(function (result) {
-        socket.emit('last-ten-messages', result.reverse())
+      // Get all messages
+
+      // If lecturer
+      // TODO
+      if (true) {
+        messagesController.getInPrioritizedOrder(lecture).then(function (result) {
+          socket.emit('all-messages-prioritized', result.reverse())
+        })
+      } else {
+        messagesController.getAllMessages(lecture).then(function (result) {
+        socket.emit('all-messages', result.reverse())
       })
```

Comment:
Is this if{}else{}, needed? Doesn't the message list get sorted automatically in the frontend by React? Especially this line:
`messagesController.getInPrioritizedOrder`
would need to be implemented and tested in the controller.